node/src/chain_spec.rs

```diff
@@ -14,102 +14,67 @@
+	/// The relay chain of the Parachain.
+	pub relay_chain: String,
+	/// The id of the Parachain.
+	pub para_id: u32,
+}
+
+impl Extensions {
+	/// Try to get the extension from the given `ChainSpec`.
+	pub fn try_get(chain_spec: &dyn sc_service::ChainSpec) -> Option<&Self> {
+		sc_chain_spec::get_extension(chain_spec.extensions())
+	}
+}
+
 type AccountPublic = <Signature as Verify>::Signer;
 
-/// Generate an account ID from seed.
+/// Helper function to generate an account ID from seed
 pub fn get_account_id_from_seed<TPublic: Public>(seed: &str) -> AccountId
 where
 	AccountPublic: From<<TPublic::Pair as Pair>::Public>,
 {
 	AccountPublic::from(get_from_seed::<TPublic>(seed)).into_account()
 }
 
-/// Generate an Aura authority key.
-pub fn authority_keys_from_seed(s: &str) -> (AuraId, GrandpaId) {
-	(get_from_seed::<AuraId>(s), get_from_seed::<GrandpaId>(s))
-}
-
-pub fn testnet_authorities() -> Vec<(AuraId, GrandpaId)> {
-	use sp_core::crypto::UncheckedInto;
-
-	// ./scripts/prepare-test-net.sh 2
-	let initial_authorities: Vec<(AccountId, AccountId, AuraId, GrandpaId)> = vec![
-		(
-			//5Dkp2xwztRtRkXdmLDJUm8wvdHWSApgYgJY3rB5brKZkWJPM
-			hex!["4ad2d854d1bc5bbfdf93b689cb1aa3986684126c2b16d86924019de6798b0971"].into(),
-			//5DvnJs3jM7sJkgfpVDxwu9GNmpFyzJSn2jG9w1hxBBWwqi92
-			hex!["526d7d3d5357d20e4cd75d1a452bd5b4903caf160f55d6bd19807efbdf165319"].into(),
-			//5GKFbTTgrVS4Vz1UWWHPqMZQNFWZtqo7H2KpCDyYhEL3aS26
-			hex!["bc09354c12c054c8f6b3da208485eacec4ac648bad348895273b37bab5a0937c"]
-				.unchecked_into(),
-			//5ErzrnqhfgXJeuFsRvwNsrbqBAbiePoqeeDZZYzaTDxwtANY
-			hex!["7bc6fd5dc6e832b294bbf2ae21df67f990a526793a9ded12a5e54e40a5a94d1d"]
-				.unchecked_into(),
-		),
-		(
-			//5Gnadix7NJH2K7Akr5hVKyXqtYqTsQJbXnKFFt274MsG8n7h
-			hex!["d0e0ece66fb861b82383e85326a2e179316021105492820ca544ea8743620b59"].into(),
-			//5FADgcPNMtcLtrmd16rZk6HJfDWqtEDT4N3jMSqtFo8tJhRr
-			hex!["88e95527362f479ebf30502db2f7d88329e034f5d77aed585042c548fa93ae01"].into(),
-			//5EPRJHm2GpABVWcwnAujcrhnrjFZyDGd5TwKFzkBoGgdRyv2
-			hex!["66be63b7bcbfb91040e5248e2d1ceb822cf219c57848c5924ffa3a1f8e67ba72"]
-				.unchecked_into(),
-			//5HNx7nq3brkA3St9CAU1KijnscvAsa5ornNuk1H1iSuXy1nd
-			hex!["eb17972691ec3a7d09a316baddc8838362ade2c12a21a506d697903e16577bfd"]
-				.unchecked_into(),
-		),
-	];
-	initial_authorities
-		.into_iter()
-		.map(|(_, _, aura_id, grandpa_id)| (aura_id, grandpa_id))
-		.collect()
-}
-
-pub fn testnet_root() -> AccountId {
-	hex!("baa78c7154c7f82d6d377177e20bcab65d327eca0086513f9964f5a0f6bdad56").into()
-}
-
-pub fn development_config() -> Result<ChainSpec, String> {
-	let wasm_binary = WASM_BINARY.ok_or_else(|| "Development wasm not available".to_string())?;
+pub fn development_config(id: ParaId) -> ChainSpec {
+	// Give your base currency a unit name and decimal places
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("tokenSymbol".into(), "CAN".into());
+	properties.insert("tokenDecimals".into(), 12.into());
```

Comment:
Ah, we want to use the relay chain currency here and not have a separate one. Does anybody know if it's sufficient to just remove the `properties` altogether?

Reply:
We need to keep the properties here and align them with the relay-chain we'll connect to.
E.g. Kusama
```
let mut properties = sc_chain_spec::Properties::new();
properties.insert("tokenSymbol".into(), "KSM".into());
properties.insert("tokenDecimals".into(), 12.into());
```

Reply:
We'll also need to do native token teleport in order to use the relay-chain token.

Reply:
This also means we do not need our own faucet but can just point users to the rococo faucet?